tests/load/README.md

```diff
@@ -219,6 +208,128 @@ Execute the `setup_k8s.sh` file from the root directory and select the **delete*
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [WAIT TIME][13]   | - Changing this cadence will increase or decrease the number of channel subscriptions and notifications sent by an AutopushUser. <br/>- This value can be set in the Locust UI.                                  |
+| [TASK WEIGHT][14] | - Changing this weight impacts the probability of a task being chosen for execution. <br/>- This value is hardcoded in the task decorators of the AutopushUser class.                                            |
+| USERS_PER_WORKER  | - This value should be set to the maximum number of users a Locust worker can support given CPU and memory constraints. <br/>- This value is hardcoded in the AutopushLoadTestShape class.                       |
+| WORKER_COUNT      | - This value is derived by dividing the total number of users needed for the performance test by the 'USERS_PER_WORKER'. <br>- This value is hardcoded in the AutopushLoadTestShape and the setup_k8s.sh script. |
+
+## Calibrating for USERS_PER_WORKER
+
+This process is used to determine the number of users that a Locust worker can support.
+
+### Setup Environment
+
+#### 1. Start a GCP Cloud Shell
+
+The load tests can be executed from the [contextual-services-test-eng cloud shell][15].
+If executing a load test for the first time, the git autopush repository will need to
+be cloned locally.
+
+#### 2. Configure the Bash Script
+
+* The `setup_k8s.sh` file, located in the `tests\load` directory, contains
+  shell commands to **create** a GKE cluster, **setup** an existing GKE cluster or
+  **delete** a GKE cluster
+    * Execute the following from the root directory, to make the file executable:
+      ```shell
+      chmod +x tests/load/setup_k8s.sh
+      ```
+
+#### 3. Create the GCP Cluster
+
+* In the `setup_k8s.sh` script, modify the `WORKER_COUNT` variable to equal `1`
+* Execute the `setup_k8s.sh` file from the root directory and select the **create** 
+  option, in order to initiate the process of creating a cluster, setting up the env 
+  variables and building the docker image
+  ```shell
+  ./tests/load/setup_k8s.sh
+  ```
+* The cluster creation process will take some time. It is considered complete, once
+  an external IP is assigned to the `locust_master` node. Monitor the assignment via
+  a watch loop:
+  ```bash
+  kubectl get svc locust-master --watch
+  ```
+
+### Calibrate 
+
+Repeat steps 1 to 3, using a process of elimination to determine the maximum 
+USERS_PER_WORKER. The load tests are considered optimized when CPU and memory resources
+are maximally utilized. This step is meant to determine the maximum user count that a
+node can accommodate by observing CPU and memory usage while steadily increasing or
+decreasing the user count.
```

Comment:
By the way, is this a place you'd want to mention that bisection formula? I think since it provides a clear way for someone to know how to +/- the number of users. Maybe just a short blurb or formula example? 

Perhaps add a little extra too that the CPU and memory can be monitored in the GCS console and the Locust UI:

```suggestion
decreasing the user count. You can monitor the CPU percentage in the Locust UI but also in the Kubernetes engine Workloads tab where both memory and CPU are visualized on charts. 
```